examples/StateForm-list-draggable.tsx

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable react/prop-types */
+
+import React from 'react';
+import HTML5Backend from 'react-dnd-html5-backend';
+import { DndProvider } from 'react-dnd';
+import Form from '../src';
+import Input from './components/Input';
+import LabelField from './components/LabelField';
+import useDraggable from './components/useDraggable';
+
+const { List, useForm } = Form;
+
+type LabelFieldProps = Parameters<typeof LabelField>[0];
+interface DraggableProps extends LabelFieldProps{
```

Comment:
`LabelFieldProps{` => `LabelFieldProps {`

Seems still missing types. Could you try to remove `.git/hooks` and reinstall `node_modules` to make `father` pre-commit hooks work. This will auto prettier the code when commit.

Reply:
Please reference to following code. The default config don't include the examples directory.
so father pre-commit don't work for it.

https://github.com/umijs/father/blob/86e4eb154c212072cd937d6a87fd91762159e069/packages/father/src/preCommit.ts#L113

Reply:
Let me add PR for this. Thanks for notice.

Reply:
I add "examples" to the regex of pre-commit by manual. after then, I get following errors by exec  "father pre-commit".

'react-dnd-html5-backend' and 'react-dnd' should only be used for example. it should be listed in devDependencies. 

![image](https://user-images.githubusercontent.com/5687399/61259115-60fdc200-a7ab-11e9-9067-675908d90fb0.png)

Reply:
Thanks for feedback. Add PR for this: https://github.com/umijs/fabric/pull/6

Reply:
Updated `@umi/fabric`, please check.